code/modules/food_and_drinks/machinery/coffeemaker.dm

```diff
@@ -200,6 +273,24 @@
 		update_appearance()
 		return TRUE //no afterattack
 
+/obj/machinery/coffeemaker/proc/can_brew()
```

Comment:
A proc named `can_brew()` should not be performing side effects like launching balloon alerts. Rename this to `try_brew()` or similar instead.

Reply:
Addressed in [1758d26](https://github.com/tgstation/tgstation/pull/70991/commits/1758d260bf38349a463fb0c4c94b19bde1396d9f)